File: MODUL.3/PRAK304_2310817120010_NurHikmah.py
bilangan = int(input('Input\n'))

if bilangan == 0:
    print("\nOutput \nNol")
elif bilangan > 0 and bilangan < 10:
    print("\nOutput \nSatuan")
elif bilangan >= 10 and bilangan < 20:
    print("\nOutput \nBelasan")
elif bilangan >= 20 and bilangan < 100:
    print("\nOutput \nPuluhan")
else:
    print("\nOutput \nAnda Menginput Melebihi Limit Bilangan")

File: MODUL.3/PRAK304_2310817120010_NurHikmah.c
#include <stdio.h>

int main()
{
    int bilangan;

    printf("Input\n");
    scanf("%d", &bilangan);

    if (bilangan == 0)
    {
        printf("\nOutput \nNol");
    }
    else if (bilangan > 0 && bilangan < 10)
    {
        printf("\nOutput \nSatuan");
    }
    else if (bilangan >= 10 && bilangan < 20)
    {
        printf("\nOutput \nBelasan");
    }
    else if (bilangan >= 20 && bilangan < 100)
    {
        printf("\nOutput \nPuluhan");
    }
    else
    {
        printf("\nOutput \nAnda Menginput Melebihi Limit Bilangan");
    }
    return 0;
}